### PR TITLE
Adding Slackin and API to Kubernetes Doc

### PR DESCRIPTION
**Pull request by stjohnjohnson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+## Services
+
+### SlackIn
+
+ - [SlackIn](https://github.com/rauchg/slackin) service
+ - [Docker](https://hub.docker.com/r/chk1/slackin/) image
+
+### API
+
+ - [Screwdriver](https;//github.com/screwdriver-cd/screwdriver) service
```
Comment (tkyi) [inline README.md]: `;` works?! :O

File: api.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        ports:
+        - containerPort: 8080
+        # Environment variables that point to your secrets in your secrets.yaml
+        env:
+          - name: DATASTORE_PLUGIN
+            value: dynamodb
+          - name: LAUNCHER_VERSION
+            value: v1.0.4
+          - name: URI
+            value: http://api.screwdriver.cd
```
Comment (tkyi) [inline api.yaml]: add a comment:
`# insert your Kubernetes <Load Balancer Ingress> here`

File: slackin.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    tier: slack
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 3000
+  selector:
+    app: screwdriver
+    tier: slack
+---
```
Comment (tkyi) [inline slackin.yaml]: Add a newline before this line for consistency

File: api.yaml (hunk 2 above)
Comment (stjohnjohnson) [inline api.yaml]: This is for us, though.

**Comment by nkatzman** [comment]
seeeems reasonable 👍

File: api.yaml (hunk 2 above)
Comment (tkyi) [inline api.yaml]: Ah okay

**Comment by tkyi** [comment]
👍

**Pull request merged**